src/duk_api_stack.c

```diff
@@ -2339,6 +2340,19 @@ DUK_EXTERNAL const char *duk_to_string(duk_context *ctx, duk_idx_t idx) {
 	return duk_require_string(ctx, idx);
 }
 
+DUK_EXTERNAL const char *duk_to_summary(duk_context *ctx, duk_idx_t idx) {
+	duk_tval *tv;
+
+	DUK_ASSERT_CTX_VALID(ctx);
+
+	idx = duk_require_normalize_index(ctx, idx);
+	tv = DUK_GET_TVAL_POSIDX(ctx, idx);
+	DUK_ASSERT(tv != NULL);
```

Comment:
Here index normalization is not necessary because the index is not reused when the value stack has changed (e.g. new values pushed). So a shorter alternative is: `tv = duk_require_tval(ctx, idx);`.